.editorconfig

```diff
@@ -14,6 +14,10 @@ indent_style = tab
 # Code files
 [{*.{cpp,h,rc,hpp}, SConstruct, SConscript}]
 indent_style = tab
+trim_trailing_whitespace = true
+cpp_new_line_before_open_brace_block = new_line
```

Comment:
How does this work with do..while loops? Our current convention is to place the braces on a new line only after a `)`.